### PR TITLE
Story/quad 115 sprint 4

### PR DESCRIPTION
**Pull request by rajataggarwal91**
Created an editable title area and editable and formatable textarea for description of about page in public website. External library used - tinyMCE (https://www.tinymce.com/)

### CODE REVIEW

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteAboutEditController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.spring.quadriga.domain.workbench.IProject;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.workbench.IRetrieveProjectManager;
+
+@Controller
+public class WebsiteAboutEditController {
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteAboutEditController.java:24]: javadoc missing

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteAboutEditController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    @Autowired
+    private IRetrieveProjectManager projectManager;
+
+    private IProject getProjectDetails(String name) throws QuadrigaStorageException {
+        return projectManager.getProjectDetailsByUnixName(name);
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteAboutEditController.java]: why do you need this method? It's just one line of code anyways.

File: Quadriga/src/main/webapp/WEB-INF/views/public/publicWebsiteEditAbout.jsp [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8"
+    pageEncoding="UTF-8"%>
+  
+  <script src="//cdn.tinymce.com/4/tinymce.min.js"></script>
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/public/publicWebsiteEditAbout.jsp]: Is there no heading for the page?

**Comment by rajataggarwal91** [comment]
All the 3 changes mentioned in the comments above were made

File: Quadriga/src/main/webapp/WEB-INF/views/public/publicWebsiteEditAbout.jsp [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/public/publicWebsiteEditAbout.jsp]: This is the jsp page for the about text settings page, right? This should be in the public folder but in the appropriate folder under the auth folder.

**Comment by rajataggarwal91** [comment]
Made the changes to directory structure. Please review.

**Pull request merged**